### ISSUE TITLE
`piSort`/`funSort` of `IUniv` should be blocked on the codomain

### ISSUE THREAD

**Issue by plt-amy** [agda/agda#6074]
... I think those are the right words in the title, at least, I haven't investigated yet. But consider:

```agda
open import Agda.Primitive.Cubical

_ : Set
_ = I → ?
```

this is a hard type error, but `I → Nat` (e.g.) has type set. So really this should be fine, with a postponed `funSort IUniv ... = Set` constraint?

**Comment by tlringer** [comment]
Oh, thank you! Is this documented anywhere? The instructions we followed didn't mention this.
